app/views/finders/_facet_collection.html.erb

```diff
@@ -9,6 +9,20 @@
     controls: "js-search-results-info"
   } if finder.show_keyword_search? %>
 
-  <%= render facet_collection.filters %>
-  <input type="submit" value="Filter results" class="button js-live-search-fallback"/>
+  <div data-module="gem-toggle" data-toggle-class="facet-toggle__content--hidden">
+    <a href="#" class="facet-toggle" data-controls="facet-wrapper" data-expanded="false" data-toggled-text="- Show fewer search options">
```

Comment:
Can we add `- Show fewer search options` and `+ Show more search options` as translation strings?